<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultWindow</class>
 <widget class="QMainWindow" name="resultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Траектория</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QChartView" name="chart_traect"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Скорость</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QChartView" name="chart_V"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Аэрод. коэфф.</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QChartView" name="chart_cxcy"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Страница</string>
     </attribute>
     <widget class="QChartView" name="chart_zones">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>699</width>
        <height>366</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Таблица</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTableWidget" name="table_all"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Результат</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Результаты баллистического проектирования</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
